client/packages/reports/src/DetailView/DetailView.tsx

```diff
@@ -82,6 +85,21 @@ export const DetailView = () => {
     setReportWithArgs(report);
   }, []);
 
+  const printReport = useCallback(() => {
+    if (report === undefined) {
+      return;
+    }
+
+    let reportArgs =
```

Comment:
```suggestion
    const reportArgs =
```

Reply:
You're not in Rustland any more ;)

Reply:
Also -- the `reportArgs` value is defined in multiple places -- you should be able to just define it once (up the top immediately below the `useUrlQuery` I would say)